clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -401,10 +401,14 @@ public boolean hasUnsent() {
         return batches;
     }
 
+    private Deque<RecordBatch> getDeque(TopicPartition tp) {
```

Comment:
Yeah, I'm aware of the convention. I chose `getDeque()` here because it paired nicely with `getOrCreateDeque()` and I couldn't come up with a way to use that convention for the latter.